tests/logger_logging/test_local_decorating.py

```diff
@@ -35,10 +35,10 @@ def set_trace_ids():
         trace.guid = "abcdefgh"
 
 
-def exercise_logging(logger):
+def exercise_logging(logger, message="C"):
```

Comment:
It would really be great if this was a fixture but since it's pre-existing it's not a must have.

Reply:
Yeah not inclined to change it with this PR, maybe in the future